jbmc/unit/pointer-analysis/custom_value_set_analysis.cpp

```diff
@@ -172,20 +172,19 @@ SCENARIO("test_value_set_analysis",
   {
     config.set_arch("none");
     config.main = "";
-    cmdlinet command_line;
 
     // This classpath is the default, but the config object
     // is global and previous unit tests may have altered it
-    command_line.set("java-cp-include-files", "CustomVSATest.class");
     config.java.classpath={"."};
-    command_line.args.push_back("pointer-analysis/CustomVSATest.jar");
 
-    register_language(new_java_bytecode_language);
     optionst options;
-    parse_java_language_options(command_line, options);
+    options.set_option("java-cp-include-files", "CustomVSATest.class");
+    std::vector<std::string> files = {"pointer-analysis/CustomVSATest.jar"};
```

Comment:
Nit pick: it may make things easier to read if just passing this directly to `initialize_goto_model` instead of adding this `files` local variable. Just less context for the reader to remember.